### PR TITLE
[fix](inverted index) Add session switch to compound inlist

### PR DESCRIPTION
**Pull request by zzzxl1993**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zzzxl1993** [comment]
run p0

**Comment by zzzxl1993** [comment]
run p0

**Comment by zzzxl1993** [comment]
run p0

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run p0

File: be/src/olap/rowset/segment_v2/segment_iterator.cpp [hunk 1]
```diff
@@ -830,9 +830,12 @@ Status SegmentIterator::_apply_index_except_leafnode_of_andnode() {
...
                           pred_type == PredicateType::GT || pred_type == PredicateType::GE ||
-                          pred_type == PredicateType::MATCH ||
-                          pred_type == PredicateType::IN_LIST ||
-                          pred_type == PredicateType::NOT_IN_LIST;
+                          pred_type == PredicateType::MATCH;
+        if (_opts.runtime_state->query_options().enable_inverted_compound_inlist) {
+            is_support |= (pred_type == PredicateType::IN_LIST ||
+                           pred_type == PredicateType::NOT_IN_LIST);
+        }
+        LOG(ERROR) << "is_support: " << is_support;
```
Comment (xiaokang) [inline be/src/olap/rowset/segment_v2/segment_iterator.cpp:838]: delete or change to DEBUG

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 2]
```diff
@@ -464,6 +464,8 @@ public class SessionVariable implements Serializable, Writable {
 
     public static final String LIMIT_ROWS_FOR_SINGLE_INSTANCE = "limit_rows_for_single_instance";
 
+    public static final String ENABLE_INVERTED_COMPOUND_INLIST = "enable_inverted_compound_inlist";
```
Comment (xiaokang) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:467]: enable_index_compound_inlist or enable_inverted_index_compound_inlist

**Review by xiaokang — APPROVED** [review]
LGTM

**Comment by zzzxl1993** [comment]
run buildall

**Pull request merged**